fireapp/lib/presentation/volunteer_list/volunteer_list.dart

```diff
@@ -36,6 +36,19 @@ class _VolunteerListState extends FireAppState<VolunteerList>
           child: (_,volunteers) {
             return Column(
               children: [
+                Container(
+                  padding: EdgeInsets.all(20.0),
+                  child: TextField(
+                    controller: viewModel.searchController,
+                    decoration: InputDecoration(
+                      hintText: 'Search',
```

Comment:
This should be a localized string (https://fire-app-2022.atlassian.net/wiki/spaces/fireapp202/pages/21561378/Internationalization)